src/containers/settings/Settings.tsx

```diff
@@ -12,17 +38,116 @@ const Settings: React.FC = () => {
+        <meta
+          name="Settings"
+          content="The user's account settings page, where they can change their profile information and password or deactivate/delete their account."
+        />
       </Helmet>
-      <div className="content-container">
-        <Title>Settings</Title>
-        <Form name="basic" onFinish={onFinish}>
-          <Form.Item>
-            <Button type="primary" htmlType="submit">
-              Delete Account
-            </Button>
-          </Form.Item>
-        </Form>
+      <div className="page-content-container">
+        <PageHeader
+          pageTitle="Account Settings"
+          pageSubtitle=""
+          subtitleColor=""
+        />
+
+        <Row>
+          <Col span={cSpan}>
+            <FormTitle>Profile</FormTitle>
+            <Form name="edit-profile" onFinish={onFinish}>
+              <Row>
+                <Col span={formHalfItemSpan}>
+                  <Form.Item name="first-name">
+                    <Input placeholder="First Name" />
+                  </Form.Item>
+                </Col>
+                <Col span={offsetSpan}></Col>
+                <Col span={formHalfItemSpan}>
+                  <Form.Item name="last-name">
+                    <Input placeholder="Last Name" />
+                  </Form.Item>
+                </Col>
+              </Row>
+              <Form.Item {...formLayout} name="username">
+                <Input placeholder="Username" />
+              </Form.Item>
+              <Form.Item {...formLayout} name="phone-number">
+                <Input placeholder="Phone Number" />
+              </Form.Item>
+
+              <FormTitle>Change Email</FormTitle>
+              <Form.Item {...formLayout} name="current-email">
+                <Input
+                  placeholder="Current Address"
+                  defaultValue="currentemail@email.com"
+                />
+              </Form.Item>
+              <Form.Item {...formLayout} name="new-email">
+                <Input placeholder="New Address" />
+              </Form.Item>
+              <Form.Item {...formLayout} name="confirm-new-email">
+                <Input placeholder="Confirm New Address" />
+              </Form.Item>
+              <Form.Item>
+                <SubmitButton type="primary" htmlType="submit">
+                  Save
+                </SubmitButton>
+              </Form.Item>
+            </Form>
+          </Col>
+
+          <Col span={cSpan}>
+            <FormTitle>Change Password</FormTitle>
+            <Form {...formLayout} name="change-password" onFinish={onFinish}>
+              <Form.Item
+                name="current-password"
+                rules={[
+                  {
+                    required: true,
+                    message: 'Please input your current password!',
+                  },
+                ]}
+              >
+                <Input placeholder="Current Password" />
+              </Form.Item>
+              <Form.Item
```

Comment:
Also not in the designs, but could you put text below the new password field saying "Your new password must be at least 8 characters". I think there's the option to put something below a field on a form field.

Reply:
I also added a rule to require having at least 8 characters for the new password input.
<img width="500" alt="UpdatedSettings" src="https://user-images.githubusercontent.com/22990100/100491170-92043b80-30ef-11eb-86fc-8a8c1cdfd08f.png"> 
<img width="500" alt="PasswordRule" src="https://user-images.githubusercontent.com/22990100/100491277-47cf8a00-30f0-11eb-8404-6dad2c24f434.png">

Reply:
Amazing!!